components/src/components/event-list/event-list.scss

```diff
@@ -0,0 +1,41 @@
+:host {
```

Comment:
Pretty much all of this styling needs to go into assets/sass/components. We want to use the Tailwind styles and colors for these as much as possible, but keep the Tailwind references out of the components themselves, and that approach (while admittedly a bit odd, given the state of the world today) lets us accomplish both.